DateTimeCompareValidator.php

```diff
@@ -62,7 +62,11 @@ class DateTimeCompareValidator extends Validator
      * - `{compareAttribute}`: the label of the attribute to be compared with
      */
     public $message;
-
+    /**
+     * @var string Date format according to DateTime::createFromFormat specification.
+     *
```

Comment:
You have an extra newline here. Try to have no new lines above or below the ```@var``` line.